### ISSUE TITLE
Deprecation warning for Ember.warn

### ISSUE THREAD

**Issue by denchen** [sir-dunxalot/ember-google-charts#20]
Receiving the following on Ember 2.5.0:

```
ember.debug.js:6249 DEPRECATION: When calling `Ember.warn` you must provide an `options` hash
as the third parameter.  `options` should include an `id` property.
[deprecation id: ember-debug.warn-options-missing]
See http://emberjs.com/deprecations/v2.x/#toc_ember-debug-function-options for more details.
```

which is in reference to this line: https://github.com/sir-dunxalot/ember-google-charts/blob/ab7ce63c394c6e0e0dd1d5d1cde872ed66971c30/addon/components/google-chart.js#L73

```
Ember.warn('You did not specify a chart type', type);
```

I'd send a PR myself, but I'm unsure as to exactly what `id` to use in the `options` property.

**Comment by sir-dunxalot** [comment]
I found [this demo](https://ember-twiddle.com/34d36b9121e017d2388f?openFiles=application.controller.js%2C), which seems to use an arbitrary ID (doesn't seem to link to an element).

Maybe it's best to change the `warn` to an `assert`?

**Comment by denchen** [comment]
Hmm, according to the docs: http://emberjs.com/api/#method_assert

    In a production build, this method is defined as an empty function (NOP). Uses of this method in
    Ember itself are stripped from the ember.prod.js build.

So it looks like it should be kept as `Ember.warn()`

**Comment by sir-dunxalot** [comment]
This was fixed by #21 and #53 will make the fix more in line with Ember's standards.
